resources/assets/js/components/Application/Review/SkillAccordion.tsx

```diff
@@ -0,0 +1,732 @@
+              ) : (
+                notApplicable
+              )}
+            </div>
+          </div>
+          <hr data-c-hr="thin(gray)" data-c-margin="top(2)" />
+        </>
+      );
+    default:
+      return null;
+  }
+};
+
+interface SkillAccordionProps {
+  skill: Skill;
+  skillLevel: string;
+  experiences: Experience[];
+  experienceSkills: ExperienceSkill[];
+}
+
+const SkillAccordion: React.FC<SkillAccordionProps> = ({
+  skill,
+  skillLevel,
+  experiences,
+  experienceSkills,
+}) => {
+  const intl = useIntl();
+  const locale = getLocale(intl.locale);
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  return (
+    <div
+      data-c-accordion=""
+      data-c-background="white(100)"
+      data-c-card=""
+      data-c-margin="bottom(.5)"
+      className={`${isExpanded && "active"}`}
+    >
+      <button
+        aria-expanded={isExpanded}
+        data-c-accordion-trigger=""
+        tabIndex={0}
+        type="button"
+        onClick={(): void => {
+          setIsExpanded(!isExpanded);
+        }}
+      >
+        <div data-c-grid="">
+          <div data-c-grid-item="base(1of1)">
+            <div data-c-padding="all(1)">
+              <div data-c-grid="middle">
+                <div data-c-grid-item="tl(3of4)">
+                  <p data-c-font-weight="bold" data-c-margin="left(1)">
+                    {localizeFieldNonNull(locale, skill, "name")}
+                  </p>
+                </div>
+                <div data-c-grid-item="tl(1of4)" data-c-align="base(left)">
+                  <span>{skillLevel}</span>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <span data-c-visibility="invisible">
+          {intl.formatMessage(accordionMessages.expand)}
+        </span>
+        {isExpanded ? (
+          <i
+            aria-hidden="true"
+            className="fas fa-angle-up"
+            data-c-colour="black"
+            data-c-accordion-remove
+          />
+        ) : (
+          <i
+            aria-hidden="true"
+            className="fas fa-angle-down"
+            data-c-colour="black"
+            data-c-accordion-add
+          />
+        )}
+      </button>
+      <div
+        aria-hidden="true"
+        data-c-accordion-content=""
+        data-c-background="gray(20)"
+        data-c-padding="bottom(2)"
+      >
+        <hr data-c-hr="thin(gray)" />
+        <div data-c-padding="lr(2)">
+          {experiences.map((experience) => {
+            const justification = getJustificationOfExperience(
+              skill,
+              experience,
+              experienceSkills,
+            );
+            return (
+              <ExperienceContent
+                key={`${experience.type}=${experience.id}-skill`}
+                intl={intl}
+                locale={locale}
+                experience={experience}
+                justification={justification}
+              />
+            );
+          })}
```

Comment:
If the experiences length is zero, we should probably have a message in the accordion telling the applicant they have no experiences attached to this skill.